### PR TITLE
Update readme and docs on versions and etc

### PR DESCRIPTION
**Pull request by yxlao**
Waiting for the viewer app screenshot.

---
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/1857)

### CODE REVIEW

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5546584&bid=18183946)

**Comment by yxlao** [comment]
Markdown -> rst preview:

image

File: README.md [hunk 1]
```diff
@@ -32,14 +34,21 @@ Core features of Open3D includes:
...
 * 3D visualization
+* Physically based rendering (PBR)
 * Available in C++ and Python
 
 For more, please visit the [Open3D documentation](http://www.open3d.org/docs).
 
+## Open3D viewer app
+
+Open3D now comes with a standalone 3D viewer app available on Ubuntu and macOS.
+Please stay tuned for Windows. You can download the viewer app
```
Comment (pvarvak) [inline README.md:45]: we want to add picture on side

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5546584&bid=18183946)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5546584&bid=18183946)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5546584&bid=18183946)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5546584&bid=18183946)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5546584&bid=18183946)

**Comment by yxlao** [comment]
image

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5546584&bid=18183946)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5546584&bid=18183946)

**Pull request merged**